### PR TITLE
Combine core/base with core/common

### PR DESCRIPTION
**Pull request by apc999**
`alluxio-core-base` is not really needed - as all modules depending on `alluxio-core-base` also depends on `alluxio-core-common`. Thus merging these two modules.

### CODE REVIEW

File: core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java [hunk 1]
```diff
@@ -667,13 +665,13 @@ public static SocketAddress getDataPortSocketAddress(WorkerNetAddress netAddress
...
    * Test if the input address is serving an Alluxio service. This method make use of the
    * gRPC protocol for performing service communication.
+   * This methods throws UnauthenticatedException if the user is not authenticated,
+   * StatusRuntimeException If the host not reachable or does not serve the given service.
    *
    * @param address the network address to ping
    * @param serviceType the Alluxio service type
    * @param conf Alluxio configuration
    * @param userState the UserState
-   * @throws UnauthenticatedException If the user is not authenticated
```
Comment (madanadit) [inline core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java:675]: why this change? I understand `StatusRuntimeException` might be remove grpc dependency?

**Review by madanadit — COMMENTED** [review]
Thanks a lot Bin! Left 1 minor comment.

File: core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java (hunk 1 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java:675]: it seems we hit some maven plugin bug that with these two `@throws`, style check will fail

**Review by madanadit — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please
